edgelet/edgelet-http/src/unix.rs

```diff
@@ -126,7 +93,7 @@ mod tests {
         let _srv = listener.for_each(move |(_socket, _addr)| Ok(()));
 
         let file_stat = stat(&path).unwrap();
-        assert_eq!(0o600, file_stat.st_mode & 0o777);
```

Comment:
Do we have unit tests for both arms of the conditional (pre-existing socket vs new socket)?

Reply:
It is still a draft. I want to discuss that change first. I don't understand why umask works  intermittently